boards/raspberrypi/rpi_5/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Delete this empty file.

Reply:
I deleted all unnecessary empty files. Thank you.